src/frontends/paddle/src/op/generate_proposals_v2.cpp

```diff
@@ -66,26 +51,31 @@ NamedOutputs generate_proposals_v2(const NodeContext& node) {
         auto transpose_order = default_opset::Constant::create(ov::element::i64, {3}, {2, 0, 1});
         auto transposed_variances = std::make_shared<default_opset::Transpose>(reshaped_variances, transpose_order);
         // auto transposed_variances = default_opset::Constant::create(ov::element::f32, {}, {2.0});
-        variances_bbox_deltas = std::make_shared<default_opset::Multiply>(single_bbox_deltas, transposed_variances);
+        variances_bbox_deltas = std::make_shared<default_opset::Multiply>(bbox_deltas, transposed_variances);
     }
 
     // generate im_info from im_scale
+    auto batch_index = default_opset::Constant::create<int64_t>(ov::element::i64, {1}, {0});
+    auto batch = std::make_shared<default_opset::Gather>(scores_shape, batch_index, gather_axis);
     auto im_scale = default_opset::Constant::create(ov::element::f32, {1}, {1.0});
-    auto im_info = std::make_shared<default_opset::Concat>(OutputVector{single_im_shape, im_scale}, 0);
+    auto im_scales = std::make_shared<default_opset::Broadcast>(im_scale, batch);
+    auto const_1 = default_opset::Constant::create<int64_t>(ov::element::i64, {1}, {1});
+    auto reshape_im_scales = std::make_shared<default_opset::Unsqueeze>(im_scales, const_1);
+    auto im_info = std::make_shared<default_opset::Concat>(OutputVector{im_shape, reshape_im_scales}, 1);
```

Comment:
It will be more simple if use 'Pad' op to add im_scale into im_info

Reply:
fixed